apollo-runtime-kotlin/src/commonTest/kotlin/com/apollographql/apollo/TestUtils.kt

```diff
@@ -0,0 +1,7 @@
+package com.apollographql.apollo
+
+import kotlinx.coroutines.CoroutineScope
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+
+expect fun <T> runBlocking(context: CoroutineContext = EmptyCoroutineContext, block: suspend CoroutineScope.() -> T): T
```

Comment:
KN doesn't provide `runBlocking` in common coroutines artifact, only in JVM and KN.